### PR TITLE
Adds new integration [robinostlund/homeassistant-svk-mimer]

### PR DESCRIPTION
**Pull request by robinostlund**
## Checklist

- [X] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [X] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [X] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [X] The actions are passing without any disabled checks in my repository.
- [X] I've added a link to the action run on my repository below in the links section.
- [X] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: https://github.com/robinostlund/homeassistant-svk-mimer/releases/tag/v0.1.5
Link to successful HACS action (without the `ignore` key):  https://github.com/robinostlund/homeassistant-svk-mimer/actions/runs/6967191872
Link to successful hassfest action (if integration): https://github.com/robinostlund/homeassistant-svk-mimer/actions/runs/6967191872

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -743,6 +743,7 @@
   "RobertD502/home-assistant-petkit",
   "RobHofmann/HomeAssistant-GreeClimateComponent",
   "RobHofmann/HomeAssistant-PhilipsAndroid2014",
+  "robinostlund/homeassistant-svk-mimer",
```
Comment (ludeeus) [inline integration:746]: This seems to be abandoned already?
No movement since this PR was created, and several PRs waiting.

Comment (robinostlund) [reply integration:746]: It is still active 😊

File: integration [hunk 2]
```diff
@@ -778,6 +778,7 @@
...
```
Comment (ludeeus) [inline integration:781]: <https://github.com/robinostlund/homeassistant-svk-mimer/blob/master/hacs.json>
This should be locked to SE, as this is only for users in that country.

<https://hacs.xyz/docs/publish/start/#hacsjson> (see `country`)

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 2 above)
Comment (robinostlund) [reply integration:781]: That is correct, i have added contry now :)

**Pull request merged**